cmd/kubelet/app/options/options.go

```diff
@@ -243,6 +247,13 @@ func ValidateKubeletFlags(f *KubeletFlags) error {
 		return fmt.Errorf("unknown 'kubernetes.io' or 'k8s.io' labels specified with --node-labels: %v\n--node-labels in the 'kubernetes.io' namespace must begin with an allowed prefix (%s) or be in the specifically allowed set (%s)", unknownLabels.List(), strings.Join(kubeletapis.KubeletLabelNamespaces(), ", "), strings.Join(kubeletapis.KubeletLabels(), ", "))
 	}
 
+	for k, v := range f.NodeExtendedResources {
+		// This is possibly a little more restrictive then necessary, but all pure integers are valid Quantities
+		if i, err := strconv.ParseInt(v, 10, 64); err != nil || i < 0 {
+			return fmt.Errorf("node extended resources must be whole 64-bit signed (but non-negative) integers. Instead we saw %s=%s", k, v)
```

Comment:
excluding the personallization "we".
"whole" numbers implies non-negative integers, possibly adjust.

proposing:

> node extended resources must be 64bit signed (but positive) integers; saw %s=%s